content/contributing/playground/openapi-example-render/index.md

```diff
@@ -13,5 +13,6 @@ weight: 999
 toc: true
 ---
 
-{{< openapi url="https://raw.githubusercontent.com/jitsuin-inc/archivist-docs/master/doc/openapi/access_policies.swagger.json" >}}
+{{< openapi url="https://raw.githubusercontent.com/honourfish/test/main/test2.json" >}}
```

Comment:
did you mean to leave this one here?

Reply:
I can update it to another one no problem

Reply:
Restored to original value